executor/write.go

```diff
@@ -1838,7 +1838,9 @@ func (e *ReplaceExec) exec(ctx context.Context, rows [][]types.Datum) (types.Dat
 		}
 		oldRow, err1 := e.Table.Row(e.ctx, h)
 		if err1 != nil {
-			return nil, errors.Trace(err1)
+			rErr := errors.Annotatef(err1, "[replace] the %dth of total %d rows, handle %d", idx+1, rowsLen, h)
+			log.Errorf("%s, row: %v", rErr.Error(), row)
```

Comment:
> 2018/08/15 20:26:52.580 write.go:1842: [error] [replace] the 2th of total 2 rows, handle 1: [kv:2]Error: key not exist, row: [{1 0 0 0 1 [] <nil>} {1 0 0 0 1 [] <nil>}]

The row format is not friendly for reading.

Reply:
This PR is very small. Why not fix it in this PR?

Reply:
Will this error be logged twice?

Reply:
The row info will be only shown in the log file.